test/e2e/kubernetes/workloads/pod-projected-svc-token.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+  - image: k8s.gcr.io/e2e-test-images/nginx:1.14-1
```

Comment:
This is a nginx test image we use upstream, which supports both Linux and Windows.